pkg/cmd/inspect/inspect.go

```diff
@@ -41,6 +41,8 @@ var (
 var (
 	defaults = []string{
 		"clusteroperator",
+		"nodes",
+		"persistentvolumes",
```

Comment:
I don't think this will work due to how args are read currently.

Reply:
it'll work as long as any args provided by the user aren't resource/name pairs

Reply:
cc @deads2k ^

Reply:
We can't assume that no one does`mg inspect --defaults namespace abc` well have to find a way to mitigate this.

Reply:
We could think about making  arguments and `--defaults` mutually exclusive

Reply:
Why would both not work?  Seems like we should be able to gather both the defaults, and whatever else the user adds on?